File: PHP/Events/PreliminaryPerformanceInfo.php
<?php
namespace LTD_API;

class PreliminaryPerformanceInfo
{
    private $performanceId;
    //date format
    private $performanceDate;
    private $minimalTicketPrice;
    private $maximalTicketPrice;
    
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getPerformanceDate()
    {
        return $this->performanceDate;
    }
    public function setPerformanceDate($performanceDate)
    {
        $this->performanceDate = $performanceDate;
    }

    public function getMinimalTicketPrice()
    {
        return $this->minimalTicketPrice;
    }
    public function setMinimalTicketPrice($minimalTicketPrice)
    {
        $this->minimalTicketPrice = $minimalTicketPrice;
    }

    public function getMaximalTicketPrice()
    {
        return $this->maximalTicketPrice;
    }
    public function setMaximalTicketPrice($maximalTicketPrice)
    {
        $this->maximalTicketPrice = $maximalTicketPrice;
    }
}

File: Java/Events/PreliminaryPerformanceInfo.java
package LTD_API;

import java.util.Date;

public class PreliminaryPerformanceInfo
{
    private int performanceId;
    private Date performanceDate;
    private decimal minimalTicketPrice;
    private decimal maximalTicketPrice;
    
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public Date getPerformanceDate()
    {
        return performanceDate;
    }
    public void setPerformanceDate(Date performanceDate)
    {
        this.performanceDate = performanceDate;
    }

    public decimal getMinimalTicketPrice()
    {
        return minimalTicketPrice;
    }
    public void setMinimalTicketPrice(decimal minimalTicketPrice)
    {
        this.minimalTicketPrice = minimalTicketPrice;
    }

    public decimal getMaximalTicketPrice()
    {
        return maximalTicketPrice;
    }
    public void setMaximalTicketPrice(decimal maximalTicketPrice)
    {
        this.maximalTicketPrice = maximalTicketPrice;
    }
}

